hw_12.php

```diff
@@ -0,0 +1,123 @@
+<?php
+
+declare(strict_types=1);
+
+error_reporting(E_ALL);
+ini_set('display_errors', '1');
+
+// Task 1
+
+function recovery() {
+    return 'Hello, World!';
+};
+
+echo recovery();
+
+// Task 2
+
+function recoveryYear($year) {
+    if ($year % 4 == 0 && $year % 100 == 0 && $year % 400 == 0) {
+        return true;
+    } else {
+        return false;
+    };
```

Comment:
на будущее, в if сразу будет bool
поэтому можно сразу сделать так
```suggestion
    return $year % 4 == 0 && $year % 100 == 0 && $year % 400 == 0;
```